util/template/client-class

```diff
@@ -1,15 +1,28 @@
 <?php
 
-declare(strict_types = 1);
+declare(strict_types=1);
+
+/**
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Elasticsearch PHP client
```

Comment:
OpenSearch client.

Keep the licensed to ES part for all existing files, but it should not be added to new ones.

Reply:
Hey @dblock, after this PR gets merged, I'll be changing **Elasticsearch PHP client** to  **OpenSearch PHP client** in all 279 files at once. This should make it easier for reviewers. Let me know if any more changes are needed for this PR to get merged. Thank you.